downgrade

```diff
@@ -299,6 +309,7 @@ main() {
 : "${DOWNGRADE_FROM_ALA:=1}"
 : "${DOWNGRADE_FROM_CACHE:=1}"
 : "${DOWNGRADE_NOSUDO:=0}"
+: "${MAXDEPTH:=1}"
```

Comment:
Please use `DOWNGRADE_MAXDEPTH`, to be consistent with the others. I don't like tools that expect un-namespaced environment variables, as there can be collisions or just confusion.

Reply:
Done.